recipes/_start.rb

```diff
@@ -41,12 +58,12 @@
   notifies :start, 'service[splunk]', :immediately
 end
 
-# Added in because the first time splunk is started using chef and systemd the .pid file is owned by root which causes issues. It is automatically correct when splunk is restarted.
-execute 'correct permissions' do
-  command "chown -RP #{node['splunk']['user']}:#{node['splunk']['group']} #{node['splunk']['home']}"
-  action :run
-  not_if { platform_family?('windows') }
-  only_if { Gem::Version.new(node['splunk']['package']['version']) >= Gem::Version.new('7.2.2') && node['platform_version'].to_i == 7 }
+# The first time splunk is started on linux using chef the .pid file is owned by root which causes issues.
```

Comment:
@som3guy How come this was added?  I tried removing it and everything still seemed to work fine, but maybe I'm just not running the same test scenario you were.

Reply:
It was added during testing the deployment of Splunk on the project I have been working on for the last couple of months in dev.

I am not certain what caused it exactly, my guess is somehow splunk was starting as the root user at some point in the chef-client run. It would no longer restart because the restart was happening as the splunk user.

It was happening on a new installation of EL 7.4 and Splunk version 7.2.6